api/config/v1alpha1/validation/validate_test.go

```diff
@@ -602,3 +644,7 @@ func TestGetEnvoyProxyComponentLevelArgs(t *testing.T) {
 		})
 	}
 }
+
+func toPointer[T any](t T) *T {
```

Comment:
Is there any good place for this kind of help function?

Reply:
yah you can use https://github.com/envoyproxy/gateway/blob/main/internal/utils/ptr/ptr.go